jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -185,6 +186,21 @@ function configure_workbench_profile() {
     fi
 }
 
+# Checks if the know_hosts is present when id_rsa is provided
+checkSSHFiles(){
+    local id_rsa=~/.ssh/id_rsa
+    local know_hosts=~/.ssh/known_hosts
+    if [ -f ${id_rsa} ]; then
+        if [ -f ${know_hosts} ]; then
+            log_info "~/.ssh/id_rsa and ~/.ssh/known_hosts provided."
+        else
+            log_error "~/.ssh/id_rsa is present but ~/.ssh/known_hosts wasn't provided."
```

Comment:
the point of this jira is to create the known_hosts manually if not there.
just a

```
if file_not_exists {
    create_file_with_right_permissions
}
```
would be enough

Reply:
@spolti  but if is enough create an empty known_hosts why is asked to the user submit their with address and public key inside instead of one empty ? https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/dm-openshift-prepare-con#githooks-prepare-proc-openshift-operator

Reply:
Tested the script witj creation of empty file and permissions, but doesn't works

"MountVolume.SetUp failed for volume "githooks-ssh-volume" : references non-existent secret key: known_hosts"

Seems the mount of secret happens before the execution of our scripts